app/Http/Controllers/GroupsController.php

```diff
@@ -33,6 +33,10 @@ public function index(Request $request)
         $filters = $request->query('filter');
         $query = $this->filter($query, $filters, Group::$indexes);
 
+        if (isset($filters['name'])) {
+            $query->where('name', 'REGEXP', $filters['name']);
```

Comment:
This reminded me of Dave's question when I was adding a `REGEXP` where filter in https://github.com/DoSomething/rogue/pull/985#discussion_r373617249 

(We ended up filtering against the built-in list of causes which wouldn't work here tho...)

Could we use SQL `LIKE` though to achieve the same filter?

Reply:
The reason I went with `REGEXP` instead of `LIKE` was to avoid any gotchas with case sensitivity (we don't want our search to be case sensitive). Maybe using the `Str::slug` like in https://github.com/DoSomething/rogue/commit/4411570e342c27a42ca63509f5fa3a260b622e59 could help avoid adding a regex to search by?

This PR sure got me thinking about the Algolia work the platforms team is taking on, as a later iteration of the Groups resource will have an additional `state` field that we'll want to filter by for group types that have thousands of groups...

Reply:
Ah ok, hmm tho I suppose if you use `Str::slug` you'll need to like split/join on the hyphens? Maybe worth just using `preg_replace` which I think was another method I tried in that PR. ([This](https://stackoverflow.com/a/659030) seems to be a highly recommended spell).

And totes, Algolia sounds like a neat way to filter down the line!

Reply:
I'll add this into #1049.

Reply:
~I suppose if you use Str::slug you'll need to like split/join on the hyphens?~ Nope, you can use a space as a second argument, Mendel, get a grip.

https://laravel.com/docs/5.5/helpers#method-str-slug

Reply:
I'm not so sure this will get us what we want. Some of these MFOL groups in #1047 have commas within the name. So if i search for "County, NJ" but run this filter through `Str::slug` -- I get a search string of `country nj` which won't match any of the rows that contain the substring `County, NJ`. It's for sure questionable whether a user would add commas into the search string at all, but we'd need to strip our results of the commas as well -- which is feeling a bit like over-engineering. I might give`preg_replace` a try instead per 75de1925df325650338bcccec570d42a3d66ea51, to allow commas within the search string.